draft-ietf-quic-transport.md

```diff
@@ -3592,6 +3594,21 @@ and will contain a CRYPTO frame with an offset matching the size of the CRYPTO
 frame sent in the first Initial packet.  Cryptographic handshake messages
 subsequent to the first do not need to fit within a single UDP datagram.
 
+
+### Abandoning Initial Packets {#discard-initial}
+
+A client stops both sending and accepting Initial packets when it sends its
```

Comment:
This draft uses packet processing, and accepting for things like "accepting" new connections, so I think processing is better here and on 3601